src/io.cpp

```diff
@@ -0,0 +1,61 @@
+#include <sstream>
+#include "io.h"
+#include <memory>
+
+using namespace SumColouring;
+
+std::unique_ptr<std::pair<int, int>> parseLine(const std::string &line) {
+    std::stringstream ss(line);
+
+    std::string temp;
+    ss >> temp;
+    int firstColumn = strtol(temp.c_str(), nullptr, 10);
+    ss >> temp;
+    int secondColumn = strtol(temp.c_str(), nullptr, 10);
+
+    return std::unique_ptr<std::pair<int, int>>(new std::pair<int, int>{firstColumn, secondColumn});
```

Comment:
pair may be returned without using pointers, and created with either initializer list `{first, second}` or function `std::make_pair(first, second)`